.github/workflows/terraform-deploy-aws.yml

```diff
@@ -100,9 +100,9 @@ jobs:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
           INFRACOST_TERRAFORM_CLOUD_TOKEN: ${{ secrets.TF_API_TOKEN }}
         with:
-          entrypoint: /scripts/ci/diff.sh # Do not change
           path: ${{ env.TFPath }}
           show_skipped: true
+          post_condition: '{"update": true}'
```

Comment:
I've set the post condition explicitly to be the `update` functionality. You can find other strategies here: https://github.com/infracost/infracost-gh-action#post_condition